api-compact.yaml

```diff
@@ -85,9 +91,11 @@ x-amazon-apigateway-integration: &apigateway_integration
   type: "http"
   requestParameters:
     integration.request.header.Authorization: "method.request.header.Authorization"
+    integration.request.header.Host: "${stageVariables.enterprise_host}"
```

Comment:
Is this helpful?  https://docs.aws.amazon.com/apigateway/api-reference/resource/integration/